testsuite/features/support/constants.rb

```diff
@@ -925,7 +927,7 @@
         opensuse_leap15_6-x86_64-backports-updates
         opensuse_leap15_6-x86_64-sle-updates
       ],
-    'leap15.5-aarch64' =>
+    'leap15.5-aarch64' => # TODO: Client tools not there anymore, product EoL, missing in UI
```

Comment:
maybe it's not EOL, they have a load of things that disappeared from UI because of publishing problems